EDA.qmd

```diff
@@ -78,7 +89,7 @@ To make the discussion easier, let's define some terms:
     Tabular data is *tidy* if each value is placed in its own "cell", each variable in its own column, and each observation in its own row.
 
 So far, all of the data that you've seen has been tidy.
-In real-life, most data isn't tidy, so we'll come back to these ideas again in Chapter \@ref(tidy-intro).
+In real-life, most data isn't tidy, so we'll come back to these ideas again in Part \@ref(tidy-intro).
```

Comment:
I expected this to be `@tidy-intro` ?

Reply:
The part landing pages (which were introductions before) are not numbered (just like before). But that presents a challenge for linking to the landing page for the part. For now I'll update this to refer to the two chapters in the tidy part but we might come back and revisit how / if we're numbering those.